Makefile

```diff
@@ -45,6 +56,13 @@ check_sample_files: ## Make sure each template in policies/templates has one sam
 check_format: ## Check that files have been formatted using opa fmt
 	@./scripts/check_format.sh
 
+.PHONY: docker_check_format
+docker_check_format: ## Check format of rego using OPA docker image
+	docker run -it --rm \
```

Comment:
Let's keep this consistent with the cloudbuild and use docker 0.15.x.

Reply:
Updated.